src/test/java/uk/gov/hmcts/reform/sscs/controller/CcdCallbackControllerTest.java

```diff
@@ -85,7 +91,12 @@
     @Before
     public void setUp() {
         controller = new CcdCallbackController(authorisationService, deserializer, dispatcher);
-        mockMvc = standaloneSetup(controller).build();
+        mockMvc = standaloneSetup(controller)
+            .setMessageConverters(new ByteArrayHttpMessageConverter(), new StringHttpMessageConverter(),
```

Comment:
Optional: I would new line all these converters to make it easier to read - same in MideventCallBack test class